### PR TITLE
Removes Pride Month

### PR DESCRIPTION
**Pull request by ghost**
No politics in my video games **REEEEEEEE**

I don't expect this to get merged but I'm going to try.

### CODE REVIEW

File: code/__DEFINES/time.dm [hunk 1]
```diff
@@ -18,7 +18,6 @@
 #define VALENTINES				"Valentine's Day"
 #define APRIL_FOOLS				"April Fool's Day"
 #define EASTER					"Easter"
```
Comment (Trilbyspaceclone) [inline code/__DEFINES/time.dm:20]:
```suggestion
#define EASTER					"Easter"
#define PRIDE_MONTH				"Pride Month"
```
Ive noticed that you misstaken removed the define for Pride Month!
Its fine misstakes happen.

Comment (ghost) [reply code/__DEFINES/time.dm:20]: Not funny. Didn't laugh.

Comment (Trilbyspaceclone) [reply code/__DEFINES/time.dm:20]: This will also make it compile!

**Comment by TheObserver-sys** [comment]
https://www.youtube.com/watch?v=zQawXvCd-fs
Apply video to PR, then close both immediately.

**Comment by silicons** [comment]
`I don't expect this to get merged`

i appreciate new contributors but 
please
c'mon

**Pull request closed without merging**